package.json

```diff
@@ -26,8 +27,9 @@
     "react-native-screens": "^2.7.0",
     "react-native-vector-icons": "^6.6.0",
     "react-native-web": "~0.11.7",
+    "react-redux": "^7.2.0",
     "redux": "^4.0.5",
-    "expo-asset": "~8.1.4"
+    "redux-thunk": "^2.3.0"
```

Comment:
Can we take a look if redux saga is compatible with react native? react-saga is so much better than redux-thunk